Changes for RabbitMQ transport v10 with ServiceControl (#7038)
Particular/docs.particular.net: servicecontrol/transports.md

* `UseExternalAuthMechanism=true|false(default)` - Specifies that an [external authentication mechanism should be used for client authentication](/transports/rabbitmq/connection-settings.md#transport-layer-security-support-external-authentication).
* `DisableRemoteCertificateValidation=true|false(default)` - Allows ServiceControl to connect to the broker [even if the remote server certificate is invalid](/transports/rabbitmq/connection-settings.md#transport-layer-security-support-remote-certificate-validation).

These options are available for only quorum queues in version 6.3 and above:

> * `ValidateDeliveryLimits=<true(default)|false>` - Validate that the delivery limit of quorum queues in RabbitMQ are set to unlimited (-1).

Reviewer: ```suggestion
* `ValidateDeliveryLimits=<true(default)|false>` - Validate that the delivery limit of each quorum queue used by NServiceBus is set to unlimited (-1).
```
